server/webhook_http.go

```diff
@@ -98,6 +98,11 @@ func (p *Plugin) httpWebhook(w http.ResponseWriter, r *http.Request, instanceID
 		return respondErr(w, http.StatusBadRequest, err)
 	}
 
+	// Skip channel posts for standalone comment events to avoid duplicates
+	if wh.ShouldSkipChannelPost() {
+		return http.StatusOK, nil
+	}
```

Comment:
3/5 that the API should not show about business logic. Can we handle that somewhere else?